2-WebApp-graph-user/2-1-Call-MSGraph/AppCreationScripts/Cleanup-WithCertificates.ps1

```diff
@@ -0,0 +1,154 @@
+﻿
```

Comment:
Did you want this PR to update 2-1 sample?

Reply:
Ah, this is accidental -will revert back

Reply:
done